hal/armv7a/pmap.c

```diff
@@ -66,6 +66,7 @@ extern unsigned int _etext;
 #define PDIR_TYPE_L2TABLE 0x00001
 #define PDIR_TYPE_INVALID 0x00000
 
+#define PDIR_ATTRS (PGHD_PRESENT | PGHD_READ | PGHD_WRITE)
```

Comment:
Nitpick - not really used for pdir, only for ptables, how about
```suggestion
#define SCRATCH_ATTRS (PGHD_PRESENT | PGHD_READ | PGHD_WRITE)
```